### ISSUE TITLE
Adjust reporting bugs page

### ISSUE THREAD

**Issue by klemmster** [klemmster/libavg_svn_migration#14]
replace textual description of how to get to issue page with actual links

https://www.libavg.de/site/projects/libavg/wiki/BugReport 

depends on #20

**Comment by klemmster** [comment]
h1. Reporting Bugs

If you've found a bug in libavg, please report it on the bug tracking system. It can be reached in the toolbar above ('Issues' and 'New Issue'). It will lead you to "github":https://www.github.com", and you'll need to sign up there.

Please report a bug if you hit a segmentation fault or an assertion, if make check fails on your machine, or if libavg is doing something else that it shouldn't be. If you think you might have found a bug but aren't sure, please ask in the forums. Also, please check the installation guides if you can't get libavg running at all - most causes of installation failures are described in the wiki.

h1. What to Include

To find out what is happening, we need to reproduce the bug. In the majority of cases, bug reports don't contain enough information for us to do this.

We always need to know:
- The libavg version: Installer, tarball, git checkout? If git checkout: which commit and branch?
- The exact program output.
- If make check terminates without errors. If there are errors in make check, please sent the output of make check. 

If it's a configuration-related bug (many are), we need to know:
- What type of system you are running libavg on (OS and exact version, processor, graphics card, driver version).
- Under Mac OS X, open the System Profiler (About This Mac -> More Info), then choose File -> Save As and RTF file format, or just tell us the version of Mac OS X as well as the model and graphics card of the computer.
- Under linux, we need to know the distribution (incl. version) and the output of:

<pre><code class="shell">
$ uname -a
$ glxinfo
</code></pre>

h1. Segmentation Faults

If you're running into a segmentation fault, a stack trace helps:

<pre><code class="shell">
 $ gdb python
 (gdb) r Test.py
 [...]
 ./bin/bash: line 4:  4324 Segmentation fault      ${dir}$tst
 (gdb) bt
</code></pre>

Replace 'Test.py' with whatever libavg program is causing the segfault. If the segfault doesn't happen directly after the 'r Test.py' line, insert a 'c' command (for 'continue'). The 'b' command should then generate your stack trace.

**Comment by klemmster** [comment]
Done
